__tests__/app.test.js

```diff
@@ -53,6 +53,75 @@ describe("GET /api", () => {
           }
         }
       });
+  })
+  it("endpoint.json contains api/articles/:articles_id", () => {
+    return request(app)
+    .get("/api")
+    .then(({body : {endpoints}}) => {
+     expect(endpoints['GET /api/articles/:article_id'])
```

Comment:
You will need to do expect(endpoints).toHaveProperty(...) rather than what you have as because you are not using a matcher it will always return true